python/lsst/pipe/tasks/characterizeImage.py

```diff
@@ -40,7 +40,42 @@
 __all__ = ["CharacterizeImageConfig", "CharacterizeImageTask"]
 
 
-class CharacterizeImageConfig(pexConfig.Config):
+class CharacterizeImageConfig(pipeBase.PipelineTaskConfig):
+    # Gen3 IO options
+    taskName = pexConfig.Field(
+        dtype=str,
+        doc="Name of task",
+        default="charImage",
+    )
```

Comment:
I don't see this config option used below, and I didn't think something like this was necessary for PipelineTasks in general.  Does it need to exist?